docs/installation/INSTALL_PATCHMATCH.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Installing PyPatchMatch
+---
+
+# :octicons-paintbrush-16: Installing PyPatchMatch
+
+pypatchmatch is a Python module for inpainting images. It is not
+needed to run InvokeAI, but it greatly improves the quality of
+inpainting and outpainting and is recommended.
+
+Unfortunately, it is a C++ optimized module and installation
+can be somewhat challenging. This guide leads you through the steps.
+
+## Windows
+
+You're in luck! On Windows platforms PyPatchMatch will install
+automatically on Windows systems with no extra intervention.
+
+## Linux
```

Comment:
Might want to note that these instructions are for Ubuntu (and I assume Debian?), where apt is available.

Reply:
I have clarified that the recipe is for Debian, Ubuntu and Mint.

Now the PR needs another approval!